### PR TITLE
Filter empty messages from results of aggregate dispatch

### PR DESCRIPTION
**Pull request by mdrachuk**
This PR implements empty filtering for the results of command handling, event reactions, and rejection reactions for aggregates. 

Empty messages cannot be returned directly but they can be created indirectly and should, therefore, be handled properly. A real-world case of empty messages falling into dispatching results are the returned tuples with empty `Optional` values.

### CODE REVIEW

**Comment by mdrachuk** [comment]
@armiol PTAL.

File: server/src/main/java/io/spine/server/aggregate/Aggregate.java [hunk 1]
```diff
@@ -122,6 +124,8 @@
                                 B extends ValidatingBuilder<S, ? extends Message.Builder>>
                 extends CommandHandlingEntity<I, S, B> {
 
+    private static final Empty EMPTY = Empty.getDefaultInstance();
```
Comment (armiol) [inline server/src/main/java/io/spine/server/aggregate/Aggregate.java]: Why this is here? It does not belong to an `Aggregate`. Can we please have a nested class for filtering? It should be a singleton. In this way this constant will go there.

File: server/src/test/proto/spine/test/aggregate/task.proto [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    string id = 1;
+}
+
+message AggTask {
+
+    AggTaskId id = 1;
+    
+    spine.core.UserId assignee = 2;
+}
+
```
Comment (armiol) [inline server/src/test/proto/spine/test/aggregate/task.proto]: This line is redundant.

**Comment by mdrachuk** [comment]
@armiol PTAL.

**Pull request merged**